### PR TITLE
Readme creation

### PR DESCRIPTION
**Pull request by mtthncls**
read me + resources folder + new base POIs to populate sample display on app loading

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+### Post '/picture' :
+
+  This aims to get the picture of the newly created POI., 
+  We use _formidable_ library to "transform" the picture date in json while storing the picture itself it in a folder with its Id (public/image/here).
+  The link between the POI and the new image is done by storing the picture ID on creation, and reusing it inside the POI creation to reach the storage folder.
+
+## Get
+
+### Get '/sample/:latitude/:longitude'
+
```
Comment (erabin05) [inline README.md:58]: when you have parameters, you can show an example with real data

File: README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  - the connection.query  getPoiInfosById complete the POI creation, and transform the result of the query so that the JSON can be used to show the newly created POI on the map in the app.
+
+### Post '/picture' :
+
+  This aims to get the picture of the newly created POI., 
+  We use _formidable_ library to "transform" the picture date in json while storing the picture itself it in a folder with its Id (public/image/here).
+  The link between the POI and the new image is done by storing the picture ID on creation, and reusing it inside the POI creation to reach the storage folder.
+
+## Get
```
Comment (erabin05) [inline README.md:55]: Show what type of data you got, what does the json will look like ?
( example : ```{
firstName : 'Etienne',
name :'Rabin'
}``` )
describe attributs

File: README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+# Database
+
+Database modelization available in the "resources" folder
+
+# Description of our roads:
+
+## Post : 
+
+### Post '/'
```
Comment (erabin05) [inline README.md:13]: what does look like the data I can send (JSON)

**Review by erabin05 — CHANGES_REQUESTED** [review]
Put examples :
- of the data you GET or the one you POST in JSON
- of the route with parameters (like a real longitude and latitude and which data you got with it)

File: README.md (hunk 1 above)
Comment (mtthncls) [reply README.md:58]: done

File: README.md (hunk 3 above)
Comment (mtthncls) [reply README.md:13]: done

File: README.md (hunk 1 above)
Comment (mtthncls) [reply README.md:58]: done

File: README.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+### Get '/filter/:keyword'
+
+it allows to get data of every POI linked with a specified second level keyword (category).
+
+
+### Get 'filterKeyword1/:keyword'
+
+it allows to get date of every POI linked with a specified first level keyword (theme)
+
```
Comment (erabin05) [inline README.md:162]: Give an example of JSON in all the routes

**Review by erabin05 — CHANGES_REQUESTED** [review]
- Example for all routes

File: README.md (hunk 4 above)
Comment (mtthncls) [reply README.md:162]: done

**Review by erabin05 — APPROVED** [review]
Ok

**Pull request merged**